src/Plugin_TEST.cc

```diff
@@ -186,7 +186,7 @@ TEST(DOMPlugin, LoadWithChildren)
   std::string pluginStr = R"(<plugin name='3D View' filename='MinimalScene'>
   <ignition-gui>
     <title>3D View</title>
-    <property type='bool' key='showTitleBar'>false</property>
```

Comment:
I think the problem is related to unrecognized `type` values, like `ignition.msgs.Boolean` in [triggered_publisher.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo6/test/worlds/triggered_publisher.sdf#L11)

Reply:
I was just fixing the test failure. With my change, the parser is now converting `false` into `0`.

I haven't touched the parser much before, so I'd appreciate some guidance on the proper fix. My assumption from #689 is that we only want to parse the `/pose@type` attributes here, but don't affect the rest, that's why I added the extra check for `"pose"` with a TODO to remove the special type handling from that function to a pose-specific function. But I suspect I may be missing some important detail.